src/component/Calculator.js

```diff
@@ -0,0 +1,26 @@
+const Calculator = () => (
+  <div className="contain">
```

Comment:
- [ ] Kindly create at least one child component for the `Calculator` component as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Create%20at%20least%201%20child%20component).